packages/web-twig/src/Resources/components/VisuallyHidden/__tests__/__fixtures__/visuallyHiddenDefault.twig

```diff
@@ -0,0 +1 @@
+<VisuallyHidden>Label</VisuallyHidden>
```

Comment:
I think we now write only one test file and show different options there. See Modal or FieldGroup.

Reply:
merged to one test, thanks